OutOfSchool/OutOfSchool.WebApi/appsettings.Development.json

```diff
@@ -12,7 +12,7 @@
   },
 
   "Elasticsearch": {
-    "EnsureIndex": true,
+    "EnsureIndex": false,
```

Comment:
Why?

Reply:
Changed it for local development puproses, sorry, will change it to "true" again)